build/yamls/antrea-ipsec.yml

```diff
@@ -371,6 +371,13 @@ spec:
         ports:
         - containerPort: 443
           protocol: TCP
+        readinessProbe:
+          failureThreshold: 5
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          tcpSocket:
```

Comment:
> A third type of liveness probe uses a TCP Socket. With this configuration, the kubelet will attempt to open a socket to your container on the specified port. If it can establish a connection, the container is considered healthy, if it can’t it is considered a failure.

Why does it use tcpSocket which checks transport layer connection but not httpGet which checks application layer response?

Reply:
> > A third type of liveness probe uses a TCP Socket. With this configuration, the kubelet will attempt to open a socket to your container on the specified port. If it can establish a connection, the container is considered healthy, if it can’t it is considered a failure.
> 
> Why does it use tcpSocket which checks transport layer connection but not httpGet which checks application layer response?

The container port uses TCP protocol. I'm not sure whether https would be supported.